sites/fast-creator/app/webpack.common.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const CopyPlugin = require("copy-webpack-plugin");
+const MonacoWebpackPlugin = require("monaco-editor-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const manifest = require("@microsoft/site-utilities/src/curated-html.json").join("");
+
+const appDir = path.resolve(__dirname, "./");
+const outDir = path.resolve(__dirname, "./www");
+
+module.exports = {
+    name: "root",
+    entry: {
+        main: path.resolve(appDir, "index.tsx"),
+        focusVisible: require.resolve(
```

Comment:
:focus-visible shipped in chromium 86, so you may not need this anymore. Just FYI :)

Reply:
Excellent! Thanks, that makes the bundle a wee bit smaller :)